pkg/deploy/deploy.go

```diff
@@ -62,18 +62,24 @@ type DeployConfigsOptions struct {
 }
 
 var (
-	lock                         sync.Mutex
-	concurrentDeploymentsLimiter *rest.ConcurrentRequestLimiter
-
+	lock    sync.Mutex
 	errSkip = errors.New("skip error")
 )
 
-func DeployForAllEnvironments(ctx context.Context, projects []project.Project, environmentClients dynatrace.EnvironmentClients, opts DeployConfigsOptions) error {
-	maxConcurrentDeployments := environment.GetEnvValueIntLog(environment.ConcurrentDeploymentsEnvKey)
-	if maxConcurrentDeployments > 0 {
-		log.Info("%s set, limiting concurrent deployments to %d", environment.ConcurrentDeploymentsEnvKey, maxConcurrentDeployments)
-		concurrentDeploymentsLimiter = rest.NewConcurrentRequestLimiter(maxConcurrentDeployments)
+type ctxDeploymentLimiter struct{}
```

Comment:
```suggestion
type ctxDeploymentLimiterKey struct{} // or deploymentLimiterKey
```

I did not find a general rule, but the convention is to use `...Key`: https://github.com/golang/go/issues/15229